metadata_service/entity/dashboard_summaries.py

```diff
@@ -0,0 +1,16 @@
+from typing import List
+
+import attr
+from amundsen_common.models.dashboard import DashboardSummary
+from marshmallow_annotations.ext.attrs import AttrsSchema
+
+
+@attr.s(auto_attribs=True, kw_only=True)
+class DashboardSummaries:
```

Comment:
should it be plural for the name or singular as we did with common repo?

Reply:
This model is actually just plural of amundsen-common's `DashboardSummary`. So I guess just stay with "DashboardSummaries" as DashboardSummary is already taken?

Reply:
maybe we could `from amundsen_common.models.dashboard import DashboardSummary as summary` ?

Reply:
just mostly felt we don't use plural anywhere else in the code base so it is not that consistent.

Reply:
Sounds good :)